packages/react-components/src/components/FileCard.tsx

```diff
@@ -99,31 +128,40 @@ export const _FileCard = (props: _FileCardProps): JSX.Element => {
   };
 
   return (
```

Comment:
Would tabbing on this component announce the file name if screen reader is on?

Reply:
Yes, the file name is announced when focus is on file card.